src/Serilog/Serilog.csproj

```diff
@@ -3,7 +3,7 @@
     <Description>Simple .NET logging with fully-structured events</Description>
     <VersionPrefix>3.0.0</VersionPrefix>
     <Authors>Serilog Contributors</Authors>
-    <TargetFrameworks>netstandard2.1;netstandard2.0;netstandard1.3;netstandard1.0;net45;net46;net47;net5.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;netstandard1.3;netstandard1.0;net45;net46;net47;net5.0;net6.0;net7.0</TargetFrameworks>
```

Comment:
There are no NET7 ifdefs (yet) so targeting net7 is generally useless (except some compiler optimizations that do not change things much IMO).

Reply:
👍  I'd leave `net7.0` everywhere else, but for core Serilog we might be better off just targeting LTS frameworks unless we can take advantage of new features. What do you think @SimonCropp ?

Reply:
@sungam3r you are correct. there is no value ATM targeting net7.0. i have gotten into the habit, of always targeting the latest, just in case any nullable attributes are added/changed.

@nblumhardt the only down side if a *very* small increase in build time and package size

Reply:
Fair enough, I'm not greatly concerned either way, let's roll with 7 here now and see how it pans out.